iframe-integration.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+  const script = document.getElementById('community-maps-builder')
+  const iframeId = script.getAttribute('data-iframe-id')
+  const iframe = document.getElementById(iframeId)?.contentWindow
+
+  if (!iframe) return // Sorry no iframe nothing
+
+  window.addEventListener("message", (event) => {
+    if (event.data.type === 'GET_URL') {
+      const queryString = window.location.search
+      const queryParams = new URLSearchParams(queryString)
+      iframe.postMessage({
+        type: 'SET_PARAMS_FROM_PARENT',
+        baseUrl: window.location.origin,
+        queryParams: {
+          mapPlace: queryParams.get('mapPlace')
+        }
+      }, '*')
+    }
+ }, false);
+})()
```

Comment:
Move this to maps builder repo and use in this repo instead.

Put the script in `/public/iframe-integration.js` and do a script to minify and copy the result to `public/iframe-integration.min.js`